lms/static/scripts/frontend_apps/components/dashboard/DashboardActivityFilters.tsx

```diff
@@ -260,7 +261,12 @@ export default function DashboardActivityFilters({
                 key={assignment.id}
                 value={`${assignment.id}`}
               >
-                {assignment.title}
+                <div className="flex flex-col gap-0.5">
+                  {assignment.title}
+                  <div className="text-grey-6 text-xs">
+                    {formatDateTime(assignment.created)}
```

Comment:
I used the existing `formatDateTime` function form simplicity, but [the designs](https://www.figma.com/design/rugQ2tONStRF3RL9UXZaGW/Search-Explorations?node-id=1894-905&t=pw0qaTseWumihtsc-0) show a simpler date format, so we may need to change this.